hack/ci/e2e-k8s.sh

```diff
@@ -237,9 +243,13 @@ run_tests() {
     printf '%s' "${fixed_coredns}" | kubectl apply -f -
   fi
 
-  # ginkgo regexes
+  # ginkgo regexes and label filter
   SKIP="${SKIP:-}"
-  FOCUS="${FOCUS:-"\\[Conformance\\]"}"
+  FOCUS="${FOCUS:-}"
+  LABEL_FILTER="${LABEL_FILTER:-}"
+  if [ -z "${SKIP}" ] && [ -z "${FOCUS}" ] && [ -z "${LABEL_FILTER}" ]; then
```

Comment:
SKIP was not influencing the defaulting of Focus before, to be backwards compatible should not this be ?
```suggestion
  if  [ -z "${FOCUS}" ] && [ -z "${LABEL_FILTER}" ]; then
```

Reply:
Yes, I think you are right. Changed.

As you were asking about SKIP, I also looked at the surrounding lines a bit more and started to wonder whether we should still use FOCUS and SKIP at all as defaults. We could get the same effect with the new label filter.

I decided against it (again?) because it's better to be conservative and to keep using what we used before. It also minimizes the changes to the script.

But I could also be convinced to be more audacious and clean this up so that FOCUS and SKIP are only set when explicitly requested. That would expedite our transition to using labels. :shrug:

Reply:
> That would expedite our transition to using labels.

better do baby steps, so we can identify clearly what change do what